client/components/homepage/Features.jsx

```diff
@@ -25,7 +27,8 @@ const Features = () => (
           <div className="col-md-6">
             <div className="icon brand-terciary"><i className="glyphicon glyphicon-facetime-video" /></div>
             <h2>Video & Voice Chat</h2>
-            <p>Enter description for this feature</p>
+            <p>Nothing beats talking a problem over face to face.</p>
+            <p>With Peer to Peer Video Chat, you can talk with your coding partner from anywhere.</p>
```

Comment:
Line 30 is a bit awkward. Try "Nothing beats talking about a problem face to face."
And I don't think Peer to Peer Video Chat needs capitalization